src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let newState = Object.assign({}, state);
+  let lastCopy = 0;
```

Comment:
```suggestion
  let lastStateIndex = -1;
```

Start with a negative index to skip the first iteration